build_tools/cmake/iree_cc_test.cmake

```
@@ -175,6 +175,12 @@ function(iree_cc_test)
     iree_add_test_environment_properties(${_NAME_PATH})
   endif(ANDROID)
 
+  if (NOT DEFINED _RULE_TIMEOUT)
+    set(_RULE_TIMEOUT 60)
+  endif()
+  
+
   list(APPEND _RULE_LABELS "${_PACKAGE_PATH}")
```

nit: trim extra newline

```suggestion
  if (NOT DEFINED _RULE_TIMEOUT)
    set(_RULE_TIMEOUT 60)
  endif()
  
  list(APPEND _RULE_LABELS "${_PACKAGE_PATH}")
```